tests/metrics/test_metric.py

```diff
@@ -154,6 +154,27 @@ def compute(self):
     assert a.compute() == 5
 
 
+def test_hash():
```

Comment:
Could this test be done directly on Metric objects ?

Reply:
@tchaton I apologize, could you elaborate? The hashing is being done on two subclasses of `Metric`, so I am unsure what precisely you mean.